packages/contracts-bedrock/test/libraries/StaticConfig.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Target contract
+import { StaticConfig } from "src/libraries/StaticConfig.sol";
+
+contract StaticConfig_Test is CommonTest {
```

Comment:
This doesn't need to be `CommonTest` as it doesn't touch any contracts

Reply:
looks like we need `CommonTest` because otherwise `assertEq` and `ffi` do not exist, which are used by the test

Reply:
See how its done here https://github.com/ethereum-optimism/optimism/blob/bcdf96abe62da2caaacb0d9571518a7b6c872a37/packages/contracts-bedrock/test/libraries/trie/MerkleTrie.t.sol#L10

Reply:
and what about `assertEq`?

Reply:
That should come from `Test`?

Reply:
ahh yes, you're right